frontend/src/components/NoMatch.tsx

```diff
@@ -0,0 +1,47 @@
+import {Box, Link} from '@cloudscape-design/components'
+import React from 'react'
+import TopBar from './TopBar'
+import errorPage from './../../public/img/error_pages_illustration.svg'
+import Image from 'next/image'
+import {useTranslation} from 'react-i18next'
+import {useLocationChangeLog} from '../navigation/useLocationChangeLog'
+
+export function NoMatch() {
+  const {t} = useTranslation()
+  useLocationChangeLog()
+
+  return (
+    <>
+      <TopBar />
```

Comment:
issue: reuse our `Layout` component for consistency (I don't think there should be an issue with showing the navigation bar here)

Also, for better placement of the content try to build the page the Cloudscape way. For instance see if [Container](https://cloudscape.design/components/container/?tabId=playground) or [ContentLayout](https://cloudscape.design/components/content-layout/?tabId=playground) can be useful here

Reply:
Going to use `Layout` component